config/pmd/ruleset.xml

```diff
@@ -15,6 +15,7 @@
   </rule>
   <rule ref="category/java/codestyle.xml">
     <exclude name="AtLeastOneConstructor" />
+    <exclude name="CommentDefaultAccessModifier" />
```

Comment:
maybe just disable it for tests (junit5 style)?

Reply:
no such parameter or i don't know how to do that specifically :(